recipe/meta.yaml

```diff
@@ -18,12 +18,18 @@ requirements:
   host:
     - hatchling
     - pip
-    - python >=3.9,<4.0
+    - python >=3.9
   run:
-    - python >=3.9,<4.0
-    - numpy >=1.23.1
-    - pandas >=1.4.3
-    - sqlalchemy >=1.4.27
+    - python >=3.9
+    - pandas >=2.0.0
+    - polars >=1.6.0
+    - sqlalchemy >=2.0.0
+    - pyarrow >=11.0.0
+  run_constrained:
+    - python-duckdb >=0.8.1
+    - duckdb-engine >=0.9.1
+    - psycopg2 >=2.9.3
+    - pyodbc >=4.0.35
```

Comment:
i adjusted the dependencies to match https://github.com/pydiverse/pydiverse.transform/blob/main/pyproject.toml

we should take care to adjust them in future PRs as well